src/components/list/list.spec.ts

```diff
@@ -164,6 +164,41 @@ describe('MdList', () => {
     });
   }));
 
+  it('should not show ripples for list items that are not links', async(() => {
+    var template = `
+        <md-list>
```

Comment:
Can you add a test for `md-nav-list`?  That's the main use case for links (generally if you have anchors, you should be using it, not `md-list`)